theano/tensor/tests/test_gc.py

```diff
@@ -87,7 +89,17 @@ def c(fn):
 
         # assert that f() didn't cause the function to grow
         # allow_gc should leave the function un-changed by calling
-        assert len_pre_f == len_post_f
+        if len_pre_f != len_post_f:
+            for i in range(len_pre_f//100):
+                p1 = pre_f[i*100:(i+1)*100]
+                p2 = post_f[i*100:(i+1)*100]
+                if p1 != p2:
+                    print(i)
+                    print("p1")
+                    print(p1)
+                    print("p2")
+                    print(p2)
+        assert len_pre_f == len_post_f, (len_pre_f, len_post_f)
```

Comment:
I'm less sure about this.  It doesn't really give any useful information to debug.